Error when installing Solidity compiler

I am on MacOS El Capitan 10.11.6. I ran the following commands:
`brew update`
`brew upgrade`
`brew install ethereum/ethereum/solidity`

It runs fine for some time but then, after the line...
`==> make install`
... I get the following output before the process exits:
`cd /tmp/solidity-20161220-20655-1mb9lww/solidity-0.4.4/libdevcore && ccache /usr/local/Homebrew/Library/Homebrew/shims/super/clang++   -DSHAREDLIB -Dsoldevcore_EXPORTS -I/tmp/solidity-20161220-20655-1mb9lww/solidity-0.4.4 -I/tmp/solidity-20161220-20655-1mb9lww/solidity-0.4.4/include -isystem /usr/local/include -I/tmp/solidity-20161220-20655-1mb9lww/solidity-0.4.4/libdevcore/..  -std=c++11 -fpermissive -DSTATICLIB -O3 -DNDEBUG -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk -mmacosx-version-min=10.11 -fPIC   -Wall -Wextra -Werror -Wno-unknown-pragmas -fPIC -fstack-protector -fstack-protector-strong -Wno-unused-function -Wno-dangling-else -o CMakeFiles/soldevcore.dir/CommonData.cpp.o -c /tmp/solidity-20161220-20655-1mb9lww/solidity-0.4.4/libdevcore/CommonData.cpp`
`/bin/sh: ccache: command not found`
`/bin/sh: ccache: command not found`
`/bin/sh: ccache: command not found`
`make[2]: *** [libdevcore/CMakeFiles/soldevcore.dir/CommonIO.cpp.o] Error 127`
`make[2]: *** Waiting for unfinished jobs....`
`make[2]: *** [libdevcore/CMakeFiles/soldevcore.dir/UTF8.cpp.o] Error 127`
`make[2]: *** [libdevcore/CMakeFiles/soldevcore.dir/SHA3.cpp.o] Error 127`
`/bin/sh: ccache: command not found`
`make[2]: *** [libdevcore/CMakeFiles/soldevcore.dir/CommonData.cpp.o] Error 127`
`make[1]: *** [libdevcore/CMakeFiles/soldevcore.dir/all] Error 2`
`make: *** [all] Error 2`

`If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/ethereum/homebrew-ethereum/issues`

I do have ccache installed, because `which ccache` says:
`/usr/local/bin/ccache`

Any help would be most appreciated. I am eager to have Solidity compiler set up. I have geth console running, but if I call `eth.getCompilers()`, I get an empty array `[]` but I want to see `["Solidity"]` there, so that I can start running some Solidity code and see how contracts work.

Thanks for your help...

--- a/cpp-ethereum.rb
+++ b/cpp-ethereum.rb
@@ -13,8 +13,6 @@
 # (c) 2014-2017 cpp-ethereum contributors.
 #------------------------------------------------------------------------------
 
-require 'formula'
-
 class CppEthereum < Formula
   version '1.3.0'
 

--- a/ethereum.rb
+++ b/ethereum.rb
@@ -1,5 +1,3 @@
-require 'formula'
-
 class Ethereum < Formula
   homepage 'https://github.com/ethereum/go-ethereum'
   url 'https://github.com/ethereum/go-ethereum.git', :tag => 'v1.8.2'

--- a/solidity.rb
+++ b/solidity.rb
@@ -27,7 +27,7 @@ class Solidity < Formula
   depends_on "gmp"
 
   def install
-    system "cmake", ".", *std_cmake_args
+    system "cmake", ".", *std_cmake_args, "-DTESTS=OFF"
     system "make", "install"
   end
 

--- a/solidity.rb
+++ b/solidity.rb
@@ -13,8 +13,6 @@
 # (c) 2014-2017 solidity contributors.
 #------------------------------------------------------------------------------
 
-require 'formula'
-
 class Solidity < Formula
 
   desc "The Solidity Contract-Oriented Programming Language"

--- a/solidity.rb
+++ b/solidity.rb
@@ -25,7 +25,7 @@ class Solidity < Formula
   depends_on "boost" => "c++11"
 
   def install
-    system "cmake", ".", *std_cmake_args, "-DTESTS=OFF"
+    system "cmake", ".", *std_cmake_args, "-DTESTS=OFF", "-DCCACHE=OFF"
     system "make", "install"
   end
 

--- a/solidity.rb
+++ b/solidity.rb
@@ -23,8 +23,6 @@ class Solidity < Formula
 
   depends_on "cmake" => :build
   depends_on "boost" => "c++11"
-  depends_on "cryptopp"
-  depends_on "gmp"
 
   def install
     system "cmake", ".", *std_cmake_args, "-DTESTS=OFF"

